### PR TITLE
Update documentaion

### LINKED ISSUE #36: Missing documentation for QueryParams and custom query
Hey,

**QueryParams**
i can't get the query with QueryParams to work and your documentation dont metnions them.
Also i guess they offer some risk for SQL-Injections?

Here is an example on what i expected to work:
```
    // Check if user already exists
    final matchingUser = (await db.users.queryUsers(const QueryParams(
      where: 'email="test@test.de"',
    )))
```

Error: (Column Â»test@test.deÂ« doesn't exist)
```
PostgreSQLSeverity.unknown 42703: Spalte Â»test@test.deÂ« existiert nicht 
_PostgreSQLExecutionContextMixin._query                  package:postgres/src/connection.dart:513
_PostgreSQLExecutionContextMixin.query                    package:postgres/src/connection.dart:474
Database.query                                                              package:stormberry/…/core/database.dart:94
```

**custom query**
I think it would be awesome if you would add a minimalistic example to the documentation whre you explain about Queries ([https://pub.dev/packages/stormberry#queries](https://pub.dev/packages/stormberry#queries))

Thanks!

### PR DESCRIPTION
**Pull request by easazade**
closes #36

### CODE REVIEW

**Comment by easazade** [comment]
feel free to change anything

**Comment by schultek** [comment]
Thanks for your effort, really appreciate it.

Ideally the example queries use query parameters to be safe against sql injections.

> Additionally to the model tables, you can query the model views to automatically get all resolved
relations without needing to do manual joins. Table names are always plural, e.g. users and view
names are in the format as complete_user_view

That is actually outdated and not true anymore. In a previous version the views were actual database views which you could query by their name. Now they are simply generated queries that are executed. You can access them using the `...Queryable` class that you already used in the example.
So this paragraph can be removed.

**Comment by easazade** [comment]
Removed that paragraph. should I change anything else?

File: doc/queries_and_actions.md [hunk 1]
```diff
@@ -3,16 +3,87 @@
...
+
+  UpdateTitleAction(this.title);
+
+  @override
+  Future<void> apply(Database db, InfoPostView request) async {
+    await db.query("""
+      UPDATE posts
+      SET title = '$title'
+      WHERE id = ${request.id}
+""");
```
Comment (schultek) [inline doc/queries_and_actions.md:60]:
```suggestion
    await db.query("""
      UPDATE posts
      SET title = @title
      WHERE id = @id
    """, {
      'title': title,
      'id': request.id,
    });
```

**Review by schultek — APPROVED** [review]

**Pull request merged**